### DISCUSSION TITLE
Please add useful/advanced search/filtering to /mails/compose

### DISCUSSION THREAD

**Discussion by fhars** [pretalx/pretalx#1233] [Ideas] [answered]
The current recipient selection in the mail editor is almost always not what I would want. By default, it sends a mail to every proposal that has at least one field in common with the proposals from the subset I actually want to write to (so, if I want to write to all accepted proposals in track X, I inadvertently end up mailing any proposal that is accepted (no matter the track) or in track X (no matter the state)). Currently, the only way to control who you are sending mail to is to open two browser windows side by side, search for the proposals in one and then manually select them all in the mail editor in the other window. This isn't made easier by the fact that I have not yet found out if there is a way to sort the proposals in the search results in the same order as in the mail editor multi select.

Showing a list of he proposals matched by the current selection would be a boon, too.

**Comment by rixx** [comment, marked as answer]
When requesting features like this, could you please add details regarding your instance and version? I would assume that this has been solved by closing #715, which is live on pretalx.com, but not in a released version yet.

The new filters are linked with "AND" instead of "OR" logic, and permit you to use placeholders including a preview feature. The feature looks like this: 
![2021-12-17T19:02+01:00](https://user-images.githubusercontent.com/2657547/146588351-46efc509-5d96-4c55-ab99-078d3db9982f.png)

**Reply by fhars** [reply]
I only looked at the _open_ tickets and saw that nothing like that was planned, this does indeed look better. And I don't know which version of pretalx I am on, there seems to be no discoverable way to find out in the UI of the instance I am using.
